stylesheets/styles.css

```diff
@@ -336,10 +336,12 @@ td {
 }
 
 .horizontal {
-  float: center;
```

Comment:
@AnkitKhatri55555 Just keep `float: none`. Keeping it in the middle doesn't look good. This is better.
![contactsysters](https://user-images.githubusercontent.com/30488330/36666853-e8dee014-1b11-11e8-8aa4-81619d4a6324.png)
What say @sara-02?